### PR TITLE
Privacy control

### PR DESCRIPTION
**Pull request by neysapatel**
Displaying other users' itineraries now. Users will only see itineraries of other users that have their privacy control settings to either "public" or to their "followers". Any user is allowed to block a user from seeing their itinerary.

### CODE REVIEW

File: app/src/main/java/com/example/whereto/fragments/SeeOthersFragment.java [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+public class SeeOthersFragment extends Fragment {
+    private SharedViewModel model;
+    private UserPreferences userPreferences;
+    protected ItineraryAdapter adapter;
+    protected List<YelpBusiness> allItineraries;
+    RecyclerView rvItinerary;
+
+    public SeeOthersFragment() {
+    }
```
Comment (FareedQ) [inline app/src/main/java/com/example/whereto/fragments/SeeOthersFragment.java:38]: Could you help me understand why this exists?

File: app/src/main/java/com/example/whereto/models/ItineraryAdapter.java [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+            tvRating.setText("" + business.getRating());
+
+            String categories = "";
+            for (YelpCategory category : business.getCategories()) {
+                categories += category.getTitle() + ", ";
+            }
+            tvCategories.setText(categories);
+
+            if (business.getImageUrl() != null) {
+                int rad = 75;
```
Comment (FareedQ) [inline app/src/main/java/com/example/whereto/models/ItineraryAdapter.java:98]: please make constant and use name radius

File: app/src/main/java/com/example/whereto/models/YelpBusiness.java [hunk 3]
```diff
@@ -13,9 +13,13 @@ public class YelpBusiness {
     private double rating;
     @SerializedName("distance")
     private double distanceAway;
+   // @SerializedName("display_address");
```
Comment (FareedQ) [inline app/src/main/java/com/example/whereto/models/YelpBusiness.java:16]: please remove commented out code

**Review by FareedQ — CHANGES_REQUESTED** [review]
A few small changes requested - Great job

File: app/src/main/java/com/example/whereto/fragments/SeeOthersFragment.java (hunk 1 above)
Comment (neysapatel) [reply app/src/main/java/com/example/whereto/fragments/SeeOthersFragment.java:40]: I added it in because GSON requires an empty constructor!

**Pull request merged**